### DISCUSSION TITLE
Daily 27/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#116] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve trabajando sobre la activity de registración, la terminé y la subí al repo. Luego  traté de agregar la ActionBar a las activities y no pude, si bien se ven en pantalla, cuando emulo en el celular no logro verlas.
Hoy continué tratando de resolver la visualización de ActionBar y finalmente pude encontrar donde estaba el error. Está terminado y subido al repo. Tengo clase de Inglés a la noche y tengo que preparar la consigna para hoy. Mas tarde estaré tratando de conectar login y registro con las activities que faltan.

**Comment by BraianTroncoso** [comment]
Ayer terminé todas las tareas asignadas, hoy cree nuevas y también las terminé. Participé de un meet en donde resolvimos cuestiones con los compañeros.
Resolví bugs, y hoy me voy a poner a ayudar con las que quedan. También sigo estudiando por mi parte para entender mejor algunos temas puntuales con android.
